### DISCUSSION TITLE
macOS Sonoma

### DISCUSSION THREAD

**Discussion by Elionis** [rustdesk/rustdesk#7420] [General]
### Bug Description

- Does not remember passwords of controlled clients
- Does not keep account and password on macOS after reboot the APP
- When i take control on client by macOS APP cannot keep configuration about keyboard and screen 100% always re-config parameters

No problem on Windows

### How to Reproduce

You config all good parameters and you reboot macOS APP and all parameter diseapper

### Expected Behavior

Keep all parameters :)

### Operating system(s) on local side and remote side

Sonoma

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![Capture d’écran 2024-03-17 à 20 25 50](https://github.com/rustdesk/rustdesk/assets/125308732/4375788d-025a-4366-be0b-53926aa876d4)

**Comment by Elionis** [comment]
Update : All Work on macOS x86-64
On ARM not work ...

**Comment by rustdesk** [comment]
i believe this is some file write permission issue

**Reply by Elionis** [reply]
Yes, i suppose too.
note : Filevault is activated too

**Comment by Elionis** [comment]
UP ?

**Comment by Elionis** [comment]
Solution found.
Just add chown on Library/Preferences/com.carriez.Rusdesk
